packages/tools/viewer/src/viewer.ts

```diff
@@ -562,6 +568,7 @@ export class Viewer implements IDisposable {
         this._autoSuspendRendering = options?.autoSuspendRendering ?? true;
         {
             const scene = new Scene(this._engine);
+            scene.clearColor = options?.clearColor ? new Color4(...options.clearColor) : new Color4(0, 0, 0, 0);

```

Comment:
TIL that you can use ... to unroll an object as parameters :D

Reply:
In this case it is a tuple but effectively yes :)